packages/at_persistence_secondary_server/pubspec.yaml

```diff
@@ -19,6 +19,10 @@ dependencies:
   at_persistence_spec: ^3.0.0
   meta: ^1.8.0
 
+dependency_overrides:
+  at_persistence_spec:
+    path: ../at_persistence_spec
+
```

Comment:
Was this for testing or meant to be left in?

Reply:
@XavierChanth see notes at the beginning of the PR description